tests/contrib/pyramid/test_pyramid.py

```diff
@@ -122,13 +123,14 @@ def test_double_unpatch(self):
             patch()
             unpatch()
             unpatch()
+            patch()
```

Comment:
weird... 

We need this because for other tests cases we can't have it be unpatched at the end of a test?

Reply:
Unfortunately yes.. Because of the autopatch tests we can't leave the library unpatched.. We'll need to address how to test the installation of instrumentation when we think about refactoring our tests.